values.yaml

```diff
@@ -94,9 +103,9 @@ headless:
 
 # ui service
 service:
-  type: NodePort
-  httpPort: 8080
-  httpsPort: 443
+  type: LoadBalancer
```

Comment:
-> NodePort

Reply:
there's an issue to access nifi when it runs in a secured cluster, accessing it with NodePort is not possible for and there might be a solution with LoadBalener.